src/schema/objects/columns.yaml

```
@@ -52,6 +52,17 @@ color:
     Hexadecimal. Label color for visualization.
   type: string
   unit: hexadecimal
+compcor:
+  name: ^[a|t|w|c]_compcor_cor_[0-9a-zA-Z]+$
```

```suggestion
  name: '[a|t|w|c]_compcor_cor_[0-9a-zA-Z]+'
```

In `formats` we decided not to use explicit anchors (see https://github.com/bids-standard/bids-specification/pull/885#discussion_r763266031), so I think it might be better just to wrap them in single quotes for YAML compatibility.

---

I think these are supposed to be `[0-9]`, as well.